### PR TITLE
ports jobs.variant to variant.no

### PR DESCRIPTION
**Pull request by mikaelbr**
Work done by @mathiasflaatt in https://github.com/varianter/variant.no/pull/93

### CODE REVIEW

File: pages/jobs/[listing].tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { getListings, getMatterFile } from 'src/jobs/utils/getLisings';
```
Comment (adriansberg) [inline pages/jobs/[listing].tsx:1]: The file `getLisings` should be called `getListings`.

File: pages/jobs/[listing].tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+export const getStaticProps: GetStaticProps<
+  JobListing,
+  { listing: string }
+> = async (context) => {
+  // This will never be empty as that path is caught by 'index.tsx' file
+  const fileName = `${context?.params?.listing}.md`;
+  const { data, content } = await getMatterFile(fileName);
+
+  //Parse DAta
```
Comment (adriansberg) [inline pages/jobs/[listing].tsx:34]: Old comment?

File: src/jobs/index.module.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+.job__listing__container:nth-child(2n) {
+  background-color: var(--color-secondary3);
+}
+
+.aspect__ratio {
+  position: relative;
+  width: calc(100% - 2rem);
+  height: 0;
+  padding-bottom: calc(56.18% - 2rem);
```
Comment (adriansberg) [inline src/jobs/index.module.css:100]: Is the percentage here an arbitrary value or a percentage for a given aspect ratio? Maybe it should have a comment if the latter.

File: src/jobs/index.tsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+        <section className={style.omVariant}>
+          <h2
+            className={style.omVariant__title}
+            data-no-animation={reducedMotion}
+          >
+            Hva står vi for?
+          </h2>
+          <article className={style.omVariant__wrapper}>
+            <div className={style.omVariant__blob}>
+              {/* TODO FIX WROK-AROUND when 2.0.7 is published  */}
```
Comment (adriansberg) [inline src/jobs/index.tsx:41]: Is this comment referring to @variant/components, which now is set to 2.0.11 in `package.json`?

File: src/jobs/index.tsx [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+              <BaseBlob
+                seed="Variant"
+                width={350}
+                height={350}
+                color={colors.colorPairs.secondary3.default.bg}
+                imageProps={{ src: require('src/jobs/images/undraw_lynx.png') }}
+                alt="Lady supporting a phone showing code"
```
Comment (adriansberg) [inline src/jobs/index.tsx:48]: Should the `alt` text be in norwegian?

**Review by adriansberg — CHANGES_REQUESTED** [review]

File: src/jobs/index.tsx (hunk 4 above)
Comment (mikaelbr) [reply src/jobs/index.tsx:41]: Don't know what this is, so I'll just remove it 😅

File: src/jobs/index.tsx [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+          </h2>
+          <article className={style.omVariant__wrapper}>
+            <div className={style.omVariant__blob}>
+              <BaseBlob
+                seed="Variant"
+                width={350}
+                height={350}
+                color={colors.colorPairs.secondary3.default.bg}
+                imageProps={{ src: require('src/jobs/images/undraw_lynx.png') }}
+                alt="Dame som holder smartelefon som viser kode"
```
Comment (adriansberg) [inline src/jobs/index.tsx:47]:
```suggestion
                alt="Dame som holder smarttelefon som viser kode"
```

**Review by adriansberg — APPROVED** [review]

**Pull request merged**